larray/core.py

```diff
@@ -10343,12 +10355,13 @@ def empty_like(array, title='', dtype=None, order='K'):
     return LArray(np.empty_like(array.data, dtype, order), array.axes, title)
 
 
+# No need to add @_check_axes_argument, `axes` argument will be checked in empty
```

Comment:
ok, so rather than "no need to add...", the comment should be something like: "we cannot use ... because an integer fill_value would be considered as an error"